lib/Controller/Api1Controller.php

```diff
@@ -1401,7 +1413,20 @@ public function importInView(int $viewId, string $path, bool $createMissingColum
 	#[RequirePermission(permission: Application::PERMISSION_MANAGE, type: Application::NODE_TYPE_TABLE, idParam: 'tableId')]
 	public function createTableShare(int $tableId, string $receiver, string $receiverType, bool $permissionRead, bool $permissionCreate, bool $permissionUpdate, bool $permissionDelete, bool $permissionManage): DataResponse {
 		try {
-			return new DataResponse($this->shareService->create($tableId, 'table', $receiver, $receiverType, $permissionRead, $permissionCreate, $permissionUpdate, $permissionDelete, $permissionManage, 0)->jsonSerialize());
+			return new DataResponse(
+				$this->shareService->create(
+					$tableId,
+					'table',
+					$receiver,
+					$receiverType,
+					$permissionRead,
+					$permissionCreate,
+					$permissionUpdate,
+					$permissionDelete,
+					$permissionManage,
+					Application::NAV_ENTRY_MODE_ALL
```

Comment:
FWIW it is not really applicable here (table share…), but have the same default may lead to less confusion in the future us's.